### ISSUE TITLE
[win arm64] Python check fails

### ISSUE THREAD

**Issue by spouliot** [unoplatform/uno.check#107]
Installing Python 3.10 from the Microsoft Store does not create the registry key that the check is looking for.

Regedit does not find any key named `PyLauncher` elsewhere...

**Comment by Soap-141** [comment]
Same issue.

**Comment by spouliot** [comment]
This seems to be fixed with 1.10.0

```
> Windows Python Installation Checkup Checkup...
  - Python is installed in C:\Windows\.
```

**Comment by Soap-141** [comment]
@spouliot And you installed Python from Windows store? I was on 1.10.0 and I'm still having the issue.

**Comment by Soap-141** [comment]
@spouliot I'm on Intel so it might not be the exact same issue, but I basically updated to Uno Check 1.10 from 1.5 and then installed Python from Windows Store and it never detected it.

**Comment by RonLar2000** [comment]
I had the same problem with 3.10 and 3.11 from Windows Store.  It would never pass the Python check.  Finally, I installed from https://www.python.org/downloads/ and included install of pylaunch.  After that uno-check completed successfully with no errors.

**Comment by Soap-141** [comment]
Fix being backported to 1.11 I think.
https://github.com/unoplatform/uno/discussions/11137
